### ISSUE TITLE
Slight mouse scrolling is not being registered

### ISSUE THREAD

**Issue by febLey** [atom/atom#13323]
### Prerequisites

* [x] Can you reproduce the problem in [safe mode](http://flight-manual.atom.io/hacking-atom/sections/debugging/#using-safe-mode)?
* [x] Are you running the [latest version of Atom](http://flight-manual.atom.io/hacking-atom/sections/debugging/#update-to-the-latest-version)?
* [x] Did you check the [debugging guide](http://flight-manual.atom.io/hacking-atom/sections/debugging/)?
* [x] Did you check the [FAQs on Discuss](https://discuss.atom.io/c/faq)?
* [x] Are you reporting to the [correct repository](https://github.com/atom/atom/blob/master/CONTRIBUTING.md#atom-and-packages)?
* [x] Did you [perform a cursory search](https://github.com/issues?q=is%3Aissue+user%3Aatom+-repo%3Aatom%2Felectron) to see if your bug or enhancement is already reported?

### Description

When I move the mouse wheel very slightly the editor window doesn't recognize those scroll movements. However in the tree-view and settings panel the scrolling works as expected, but in the editor window the scrolling only works if i scroll faster. The scrolling behaviour in the tree-view and settings panel is the same as in the rest of the system (like in the Finder, Chrome, etc.)

I've recorded a video to demo it:

[www.youtube.com/watch?v=DA6kkavus5E](https://www.youtube.com/watch?v=DA6kkavus5E)

I know that there's a setting for the scroll sensivity and I've played around with the value, but it didn't help.

We've tested it on 3 Mac System which all do have this problem and on a Linux machine, where it seems to work fine. The Linux machine has the same mouse as one of the the Mac systems.

### Steps to Reproduce

1. Open Atom
2. Open a project with long files and a large filetree
3. Move the mousewheel very slightly in the editor view and compare it to the scrolling in the tree-view

**Expected behavior:** The scrolling should be similar in every view.

**Actual behavior:** The scrolling in the editor view differs from the other views.

### Versions

```
OS      : macOS 10.12.1
Atom    : 1.12.5
Electron: 1.3.9
Chrome  : 52.0.2743.82
Node    : 6.5.0
```

**Comment by 50Wliu** [comment]
Thanks for the issue!  I noticed that in the video, Atom was not in safe mode.  Are you certain that you tested this behavior using `atom --safe`?

**Comment by febLey** [comment]
Yes I did, obviously not in the video ;)

**Comment by febLey** [comment]
Tried it again in safe mode, still not working :(

https://www.youtube.com/watch?v=-7xIu2x4hR0

**Comment by Policy51** [comment]
Attempted to recreate this on bug whilst using windows 7, didn't have any problems.

**Comment by febLey** [comment]
Seems to be fixed in 1.13.0. Can't recreate this behaviour.
